diff -ur /tmp/ubuntu-iso/isolinux/isolinux.cfg /srv/scratch/ubuntu-iso-modified//isolinux/isolinux.cfg
--- /tmp/ubuntu-iso/isolinux/isolinux.cfg	2011-04-26 08:57:25.000000000 -0400
+++ /srv/scratch/ubuntu-iso-modified//isolinux/isolinux.cfg	2011-10-06 22:58:59.460439017 -0400
@@ -2,5 +2,5 @@
 include menu.cfg
 default vesamenu.c32
 prompt 0
-timeout 0
+timeout 1
 ui gfxboot bootlogo
diff -ur /tmp/ubuntu-iso/isolinux/txt.cfg /srv/scratch/ubuntu-iso-modified//isolinux/txt.cfg
--- /tmp/ubuntu-iso/isolinux/txt.cfg	2011-04-26 08:57:25.000000000 -0400
+++ /srv/scratch/ubuntu-iso-modified//isolinux/txt.cfg	2011-10-06 22:58:11.860439038 -0400
@@ -1,6 +1,10 @@
-default install
+default automated
+label automated
+  menu label ^Install Ubuntu (automated)
+  kernel /install/vmlinuz
+  append auto=true priority=critical file=/cdrom/preseed/custom.seed vga=788 initrd=/install/initrd.gz text --
 label install
-  menu label ^Install Ubuntu
+  menu label ^Install Ubuntu (manual)
   kernel /install/vmlinuz
   append  file=/cdrom/preseed/ubuntu.seed vga=788 initrd=/install/initrd.gz quiet --
 label check
